src/app/team-management/page.tsx

```
@@ -0,0 +1,1358 @@
+                          <button
+                            type="button"
+                            onClick={() => openSkillModal(member)}
+                            className="text-gray-500 hover:text-gray-700"
+                            title="スキル情報を編集"
+                          >
+                            <FiFile className="h-4 w-4" />
+                          </button>
+                          <button
+                            type="button"
+                            onClick={() => openEditModal(member)}
+                            className="text-gray-500 hover:text-gray-700"
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)